### ISSUE TITLE
Monokai Pro for Visual Studio Code and PayPal

### ISSUE THREAD

**Issue by jonathanlinat** [Monokai/monokai-pro-vscode#30]
Hey there,

I'd like to purchase **Monokai Pro for Visual Studio Code** but I don't want to create a PayPal account (for many personal reasons) in order to do so.

![image](https://user-images.githubusercontent.com/14064112/31771513-cbb52540-b4b2-11e7-9df5-89a2f41a8084.png)

Isn't it possible to **activate a classic credit card form** in FatFreeCartPro?

Regards.

**Comment by sztomi** [comment]
I'd like to second this. Paypal is dishonest and actively hostile towards its users.

**Comment by joshbeitler** [comment]
Any update on this? I would really prefer to not support PayPal

**Comment by sztomi** [comment]
@Monokai would you comment on this please? If this is not on the radar I'll have to switch to something else (which is a shame because this theme is really nice and I'd really like to buy it).

**Comment by ntwb** [comment]
Duplicate of https://github.com/Monokai/monokai-pro-sublime-text/issues/18

> tl;dr: Monokai wrote:
> > _"I believe you can check out as guest with Paypal. I'm afraid that's the only option for now with my current shopping cart provider."_

I've just confirmed this works, after clicking "Buy License" on the PayPal page click "Pay with Debit or Credit Card" and enter your credit card details

**Comment by jonathanlinat** [comment]
Nope. You can't. Only two options: **Login** or **Create an account and Pay**.

![image](https://user-images.githubusercontent.com/14064112/34436929-f4ff05d0-ec78-11e7-9d15-d8cca503d4df.png)

**Comment by sztomi** [comment]
I'm seeing the same. Also, this isn't really a solution. The point is *not* avoiding account creation at Paypal. It's about avoiding supporting the shady business practices Paypal employs by not giving them money. So until this changes, no monokai purchase for me (sadly).

**Comment by ntwb** [comment]
> Nope. You can't. Only two options: **Login** or **Create an account and Pay**.

That's not what it looks like here in Australia :( Even which I switch the form to US I don't see what your screenshot shows, it's the same as this one /shrug 

![screen shot 2017-12-31 at 3 40 57 pm](https://user-images.githubusercontent.com/1016458/34459318-3e4940ec-ee41-11e7-85d9-39995ac11239.png)
